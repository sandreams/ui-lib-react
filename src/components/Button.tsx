import React from 'react'
const Button: React.FC = () => {
  return (
    <div>
      <button>按钮</button>
    </div>
  )
}
export default Button
