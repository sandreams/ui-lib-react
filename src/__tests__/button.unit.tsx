/* eslint-disable no-undef */
import renderer from 'react-test-renderer'
import { Button } from '../components/Button'
// const add = (a: number, b: number) => {
//   return a + b
// }

describe('快照测试', () => {
  it('rendered correctly', () => {
    const tree = renderer.create(<Button />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
