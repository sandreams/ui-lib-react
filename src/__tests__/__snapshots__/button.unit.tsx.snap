// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`快照测试 rendered correctly 1`] = `
<div>
  <button>
    按钮
  </button>
</div>
`;
